File: android/measure/src/main/java/sh/measure/android/tracing/SpanStatus.kt
package sh.measure.android.tracing

/**
 * Specifies the status of the operation for which the span has been created.
 *
 * @param value The value to use when marshalling this enum.
 */
enum class SpanStatus(val value: Int) {
    /**
     * Default value for all spans.
     */
    Unset(0),

    /**
     * The operation completed successfully.
     */
    Ok(1),

    /**
     * The operation ended in a failure.
     */
    Error(2),
}

File: flutter/packages/measure_flutter/lib/src/tracing/span_status.dart
/// Specifies the status of the operation for which the span has been created.
enum SpanStatus {
  /// Default value for all spans.
  unset(0),

  /// The operation completed successfully.
  ok(1),

  /// The operation ended in a failure.
  error(2);

  /// The value to use when marshalling this enum.
  const SpanStatus(this.value);

  /// The integer value associated with this status.
  final int value;
}
